### PR TITLE
Create tabs components

### PR DESCRIPTION
**Pull request by JakeHildy**
@tlouth19 this adds the Tabs component to admin-ui. 

I tried a slightly different approach by passing in an activeClassName prop to the Tab component so that it can be targeted this way. Is this closer to how you think it should be done? I was looking up the React.cloneElement, but to me it makes more sense to use that on the "other side" when you are writing a component to wrap it as a way of customizing the props... Let me know your thoughts.

I found this article on using React.cloneElement to create a NavLink in Next.js: 
https://medium.com/@mikegajdos81/how-to-create-active-navlink-in-nextjs-d6d3934bd59a

Still need to link it up and test with Admin so putting this as a draft for now.

### CODE REVIEW

**Comment by JakeHildy** [comment]
Or do we want something more like this:

Screen Shot 2022-02-23 at 4 39 12 PM

File: src/tabs/Tab.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  return (
+    <TabStyles
+      as={as}
+      onClick={onClick}
+      className={className}
+      href={href}
+      data-testid={dataTestId}
+      $activeClassName={activeClassName}
```
Comment (JakeHildy) [inline src/tabs/Tab.js:42]: @tlouth19  This is a transient prop. Not sure if you have seen it before but the "$" makes it so Styled Components doesn't add it to the DOM and its just used internally. This fixes the warning where the browser doesn't want a camelCase attribute on the DOM.

Comment (JakeHildy) [reply src/tabs/Tab.js:42]: https://stackoverflow.com/questions/57586654/styled-component-attrs-react-does-not-recognize-prop

File: src/tabs/Tabs.styles.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+const TabsStyles = styled.nav`
+  display: flex;
+  align-items: stretch;
+  column-gap: 4px;
+  padding: 0 2.5px;
+  justify-content: flex-start;
+  border-bottom: 1px solid ${({ theme }) => theme.c7__ui.borderColor};
+  margin: 0 -20px;
```
Comment (tlouth19) [inline src/tabs/Tabs.styles.js:10]: I think we should build this without the negative margin - we need it in admin but if another app uses this its going to get out of the layout. Maybe we should just do margin 0 and override it in admin? For this and the TabBody below

Looks good other than this!

**Review by tlouth19 — APPROVED** [review]

**Pull request merged**